calcom/cal.com
# feat: Auto accept on org verify

## What does this PR do?
Fixes #10005

Loom: https://www.loom.com/share/e271813e06af43e69cf36e99e54173d0

How to test: 
Create a org - add a bunch of members from your domain + other domains
Add a child team - add some members to that also

In a new window login as an account with ADMIN (root level perms not team)
Goto - settings>admin>organization
Verify a domain

All users matching the domain should auto be accepted
Login as one of the test users -> Test they are apart of the organization and that they get presented organization in settings
